shared-ui/src/commonMain/kotlin/co.touchlab.droidcon/viewmodel/session/SessionDetailScrollStateStorage.kt

```diff
@@ -0,0 +1,24 @@
+package co.touchlab.droidcon.viewmodel.session
+
+import kotlinx.datetime.LocalDate
+
+object SessionDetailScrollStateStorage {
```

Comment:
Ideally this would be part of the viewModel and not saved globally, but I'm guessing we are losing state in the viewModel some how? I think if we want to keep it global like this, we should move it to a `single` dependency that gets passed in by koin

Reply:
Yes, the view models get recreated when the data from the DB change (for example changing the rsvp). I like the idea of injecting it by koin.